app/talk/discussion-new-form.cjsx

```diff
@@ -73,6 +76,9 @@ module.exports = createReactClass
       .then (discussion) =>
         @setState loading: false
         titleInput.value = ''
+        @context?.geordi?.logEvent
+          type: 'add-discussion'
+          data: {discussion: discussion.title}
```

Comment:
unsure if preferred structure, alternatively could be something like 'data: {title: discussion.title}` or something else

Reply:
just send it either as `data: discussion.title` or else `relatedID: discussion.title`